<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="DrawPanelWidget" name="drawPanelWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>646</width>
         <height>469</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <widget class="QMenu" name="menuMode">
     <property name="title">
      <string>Mode</string>
     </property>
     <addaction name="actionReplace"/>
     <addaction name="actionXOR"/>
     <addaction name="separator"/>
     <addaction name="actionImpact"/>
    </widget>
    <addaction name="menuMode"/>
    <addaction name="actionProperties"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="actionNext"/>
    <addaction name="actionRun"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionToolbar"/>
    <addaction name="actionStatus_Bar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGame"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionReplace"/>
   <addaction name="actionXOR"/>
   <addaction name="actionImpact"/>
   <addaction name="actionProperties"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
   <addaction name="actionNext"/>
   <addaction name="actionRun"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>Create a new document</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open an existing document</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save the active document</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As..</string>
   </property>
   <property name="toolTip">
    <string>Save As..</string>
   </property>
   <property name="statusTip">
    <string>Save the active document with new name</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Quit the application; prompts to save documents</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="statusTip">
    <string>Set replace paint mode</string>
   </property>
  </action>
  <action name="actionXOR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>XOR</string>
   </property>
   <property name="statusTip">
    <string>Set XOR paint mode</string>
   </property>
  </action>
  <action name="actionImpact">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Impact</string>
   </property>
   <property name="statusTip">
    <string>Display impact values for each cell</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
   <property name="statusTip">
    <string>Change properties</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="statusTip">
    <string>Clear field</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="statusTip">
    <string>Next step</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="statusTip">
    <string>Run game</string>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar</string>
   </property>
   <property name="statusTip">
    <string>Show or hide the toolbar</string>
   </property>
  </action>
  <action name="actionStatus_Bar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status Bar</string>
   </property>
   <property name="statusTip">
    <string>Show or hide the status bar</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>Display information</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawPanelWidget</class>
   <extends>QWidget</extends>
   <header>drawpanelwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>about()</slot>
 </slots>
</ui>
